lib/galaxy/model/migrate/versions/0152_add_metadata_file_uuid.py

```diff
@@ -20,13 +21,8 @@ def upgrade(migrate_engine):
     metadata.reflect()
 
     metadata_file_table = Table("metadata_file", metadata, autoload=True)
-
-    try:
-        uuid_column = Column('uuid', UUIDType())
-        uuid_column.create(metadata_file_table)
-        assert uuid_column is metadata_file_table.c.uuid
-    except Exception:
-        log.exception("Adding column 'uuid' to `MetadataFile` table failed.")
+    uuid_column = Column('uuid', UUIDType())
+    add_column(uuid_column, metadata_file_table)
```

Comment:
```suggestion
    add_column(uuid_column, 'metadata_file', metadata)
```

and drop line 23. Similarly for `drop_column` and migration 153.

Reply:
Done, thanks for the catch. Very nice!